tutorials/math/bit-manipulation.md

```diff
@@ -66,7 +68,9 @@ n     n     n - 1  n & (n - 1)
 
 #### Example #1: [0231 - Power of Two (Easy)](https://leetcode.com/problems/power-of-two/)
 
-We know that a power of 2 is a positive number and only has one bit set. We can use $n \& (n - 1)$ to see the result is 0 or not to determine if the target value is a power of 2 or not.
+We know that a power of 2 is a positive number and only has one bit set. We can use $n$ &amp; $(n - 1)$ to see the result is 0 or not to determine if the target value is a power of 2 or not. 
+
+In short, $n$ &amp; $(n - 1)$ never have a 1 bit in the same place.
```

Comment:
What does it mean?

Reply:
For example, 1000 (8), and 0111(3) - never have 1 bit in the same place for **Power of Two**

Reply:
Add it there as well.